test/repl.jl

```diff
@@ -348,7 +348,7 @@ begin
     LineEdit.accept_result(s, histp)
     @test LineEdit.mode(s) == repl_mode
     @test buffercontents(LineEdit.buffer(s)) == "shell"
-    @test position(LineEdit.buffer(s)) == 4
+    @test position(LineEdit.buffer(s)) == 2
```

Comment:
it's testing that the cursor is on the first `l` (byte 4 of "shell") after searching backwards for it